### PR TITLE
Recommend cnp.import_array in contribution guide

### PR DESCRIPTION
**Pull request by lagru**
Sorry, I missed #4592 and thus failed to mention it there in time. Nevertheless I think we can help contributors (and ourselves) by recommending a call to `cnp.import_array()` in the contribution guide.

See also #4577.

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Comment by sciunto** [comment]
Thank you.

File: CONTRIBUTING.txt [hunk 1]
```diff
@@ -245,7 +245,9 @@ Stylistic Guidelines
    import matplotlib.pyplot as plt
    from scipy import ndimage as ndi
 
-   cimport numpy as cnp  # in Cython code
+   # only in Cython code
+   cimport numpy as cnp
+   cnp.import_arrays()
```
Comment (soupault) [inline CONTRIBUTING.txt:250]: Is this supposed to be `arrays` and not `array`? :thinking:

**Comment by sciunto** [comment]
Oh, you're correct @soupault !

**Comment by lagru** [comment]
I really don't get how that slipped me. :see_no_evil: Amazing catch @soupault!

**Pull request merged**